circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/goblin_translator_mini.fuzzer.cpp

```diff
@@ -0,0 +1,44 @@
+#include "barretenberg/numeric/uint256/uint256.hpp"
+#include "goblin_translator_circuit_builder.hpp"
+using Fr = ::curve::BN254::ScalarField;
+using Fq = ::curve::BN254::BaseField;
+
+extern "C" int LLVMFuzzerTestOneInput(const unsigned char* data, size_t size)
```

Comment:
Created a mini-fuzzer which explored possible values and quickly found the assert issue.